### PR TITLE
refactor(example): Add more usage of derived data

### PR DESCRIPTION
**Pull request by mgred**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Currently derived data does already exist but is not used in the example app.

Issue Number: #1199 

## What is the new behavior?

There should be good examples of using derived data in the example app

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by mgred** [comment]
@timdeschryver, sure I can do! These changes were made by `prettier` (precommit hook).

File: projects/example-app/src/app/auth/reducers/index.ts [hunk 1]
```diff
@@ -27,11 +29,11 @@ export const selectAuthStatusState = createSelector(
   selectAuthState,
   (state: AuthState) => state.status
 );
+export const getUser = createSelector(selectAuthStatusState, fromAuth.getUser);
 export const getLoggedIn = createSelector(
-  selectAuthStatusState,
-  fromAuth.getLoggedIn
+  getUser,
+  (user: User | null) => !!user
```
Comment (timdeschryver) [inline projects/example-app/src/app/auth/reducers/index.ts]: The typings here aren't needed, it already knows that `user` is `User | null`.

Comment (mgred) [reply projects/example-app/src/app/auth/reducers/index.ts]: Ok, I'll change this. Thanks

Comment (timdeschryver) [reply projects/example-app/src/app/auth/reducers/index.ts]: You can also remove the `User` import now.

Comment (mgred) [reply projects/example-app/src/app/auth/reducers/index.ts]: so true :+1:

**Pull request merged**